src/app/app.component.html

```diff
@@ -884,43 +890,6 @@ <h4 style="text-align: center">
         >
         </ngx-charts-bubble-chart-interactive>
       </div>
-      <ngx-charts-line-chart
```

Comment:
Removed this extra line-chart and added the reference lines to the existing "normal" line-chart. Hope, that is ok 😇?